### PR TITLE
Remove md5 twig extension

### PR DESCRIPTION
**Pull request by laps15**
Adapt templates to remove md5 from PhpFuncionsExtension.php

Signed-off-by: Leonardo Strozzi <laps15@inf.ufpr.br>

Before submitting pull request, please check that every commit:

- [x] Has proper Signed-Off-By
- [x] Has commit message which describes it
- [x] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [x] Any new functionality is covered by tests

### CODE REVIEW

File: libraries/classes/Controllers/Database/DatabaseStructureController.php [hunk 1]
```diff
@@ -654,6 +654,8 @@ protected function displayTableList()
                 Template::get('database/structure/structure_table_row')
                     ->render(
                         array(
+                            'table_name_hash'       => md5($current_table['TABLE_NAME']),
+                            'db_table_name_hash'       => md5($this->db.$current_table['TABLE_NAME']),
```
[the remark this answers is not in the document]
Comment (laps15) [reply libraries/classes/Controllers/Database/DatabaseStructureController.php]: Done! There was a '.' missing between the db name ante the table name, fixed that too.

File: templates/table/relation/internal_relational_row.twig [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <tr>
     <td class="vmiddle">
+        <h1>{{ myfield_md5 }}</h1>
```
[the remark this answers is not in the document]
Comment (laps15) [reply templates/table/relation/internal_relational_row.twig]: Totally unintentional, I was using for debug only.

File: templates/table/relation/common_form.twig [hunk 3]
```diff
@@ -117,7 +117,7 @@
                         {% set myfield = save_row[i]['Field'] %}
                         {# Use an md5 as array index to avoid having special characters
                            in the name attribute (see bug #1746964 ) #}
-                        {% set myfield_md5 = md5(myfield) %}
+                        {% set myfield_md5 = save_row[i]['Hash'] %}
```
[the remark this answers is not in the document]
Comment (laps15) [reply templates/table/relation/common_form.twig]: Fixed it, it really was empty. Now I'm using the other array (column_array).

**Comment by mauriciofauth** [comment]
Merged, thanks for your contribution!

**Comment by williamdes** [comment]
@laps15 You have maybe introduced a bug.
https://demo.phpmyadmin.net/master-config/tbl_relation.php?db=menagerie&table=event
Screenshot:
selection_027

**Pull request merged**